pkg/services/object_manager/placement/traverser.go

```diff
@@ -66,59 +67,106 @@ func NewTraverser(opts ...Option) (*Traverser, error) {
 	}
 
 	if cfg.builder == nil {
-		return nil, errors.Wrap(errNilBuilder, "incomplete traverser options")
+		return nil, errors.Wrap(errNilBuilder, invalidOptsMsg)
+	} else if cfg.policy == nil {
+		return nil, errors.Wrap(errNilPolicy, invalidOptsMsg)
 	}
 
 	ns, err := cfg.builder.BuildPlacement(cfg.addr, cfg.policy)
 	if err != nil {
 		return nil, errors.Wrap(err, "could not build placement")
 	}
 
+	ss := cfg.policy.GetSelectors()
```

Comment:
if `cnt` is a required amount of copies to place in one vector (replica), then it should be `ss := cfg.policy.GetReplicas()` and then `cnt = int(ss[i].GetCount())`